src/algebra/group/conj.lean

```diff
@@ -45,8 +68,118 @@ calc is_conj a 1 ↔ is_conj 1 a : ⟨is_conj_symm, is_conj_symm⟩
 lemma conj_injective {x : α} : function.injective (λ (g : α), x * g * x⁻¹) :=
 (mul_aut.conj x).injective
 
-@[simp] lemma is_conj_iff_eq {α : Type*} [comm_group α] {a b : α} : is_conj a b ↔ a = b :=
-⟨λ ⟨c, hc⟩, by rw [← hc, mul_right_comm, mul_inv_self, one_mul], λ h, by rw h⟩
+end group
 
-protected lemma monoid_hom.map_is_conj (f : α →* β) {a b : α} : is_conj a b → is_conj (f a) (f b)
-| ⟨c, hc⟩ := ⟨f c, by rw [← f.map_mul, ← f.map_inv, ← f.map_mul, hc]⟩
+namespace is_conj
+
+/-- The setoid of the relation `is_conj` iff there is a unit `u` such that `u * x = y * u` -/
+protected def setoid (α : Type*) [monoid α] : setoid α :=
+{ r := is_conj, iseqv := ⟨is_conj_refl, λa b, is_conj_symm, λa b c, is_conj_trans⟩ }
+
+end is_conj
+
+local attribute [instance] is_conj.setoid
```

Comment:
I think this API might be easier to use if you didn't do this, and used `quotient.mk'` and other primed quotient definitions instead; but I'm not 100% sure.

Reply:
FWIW, I'm just copying what was done for `associates`.

Reply:
Ah ok - it might be good to add a comment to indicate that near this line then, something like
```
/- This small quotient API is largely copied from the API of `associates`;
where possible, try to keep them in sync -/
```